config/config.go

```diff
@@ -307,6 +308,14 @@ func (cfg Config) ValidReferer(reqURL, ref string) bool {
 	return false
 }
 
+// ApplicationName will return the General.ApplicationName
+func (cfg Config) ApplicationName() string {
+	if cfg.General.ApplicationName == "" {
+		cfg.General.ApplicationName = "GoAlert"
```

Comment:
Just return instead of assigning to the struct
```suggestion
		return "GoAlert"
```